README.markdown

```diff
@@ -151,3 +151,8 @@ maintained by the RSpec team but may receive support and maintenance as needs re
 * [https://github.com/rspec/rspec-collection_matchers](https://github.com/rspec/rspec-collection_matchers)
 * [https://github.com/rspec/rspec-its](https://github.com/rspec/rspec-its)
 * [https://github.com/rspec/rspec-legacy_formatters](https://github.com/rspec/rspec-legacy_formatters)
+
+## Ruby Engine Support
+* MRI is officially supported by RSpec.
+* TruffleRuby is not officially supported by RSpec and any issues with this engine should be reported to [https://github.com/oracle/truffleruby/issues](https://github.com/oracle/truffleruby/issues).
```

Comment:
How about changing the wording here to something like:

## Ruby Engine Support
* Currently only MRI and JRuby releases are officially supported by RSpec.
* Running RSpec on other Ruby implementations might work, however issues with those implementations should be reported on their issue tracker.

I'm not comfortable with mentioning TruffleRuby as the only "non supported" implementation here. I'm sure there are a lot more Ruby implementations that can run (some of) RSpec and yet we don't need a disclaimer for each of them.
And I think `allow_failures` in `.travis.yml` makes it clear it's not officially supported anyway.

Reply:
Sounds good, I've updated the wording there.